treeherder/intermittents_commenter/commenter.py

```diff
@@ -368,13 +369,15 @@ def build_bug_map(self, bugs, option_collection_map):
         bug_map = dict()
         for bug in bugs:
             platform = bug["job__machine_platform__platform"]
+            platform = platform.replace("-shippable", "")
+            architecture = bug["job__machine_platform__architecture"]
             repo = bug["job__repository__name"]
             bug_id = bug["bug_id"]
             build_type = option_collection_map.get(
                 bug["job__option_collection_hash"], "unknown build"
             )
             test_variant = self.get_test_variant(bug["job__signature__job_type_name"])
-            platform_and_build = f"{platform}/{build_type}"
+            platform_and_build = f"{platform}/{architecture}/{build_type}"
```

Comment:
I think it would be better if you had `f"{platform}-{architecture}/{build_type}"`

in the end I think this will shift, so it will probably be ok for a while to leave this as is.  Do you have any concerns with my suggestion?